src/components/ControlBar.js

```diff
@@ -14,11 +14,17 @@ const controlBarStyle = {
   justifyContent: 'space-evenly',
 };
 
-const ConnectedControlBar = (props): React.MixedElement => {
+const ControlBar = (): React.MixedElement => {
+  const modalState = useSelector(state => state.displayModal);
+  const dispatch = useDispatch();
+
   const buttonClicked = (e: SyntheticEvent<HTMLButtonElement>): void => {
-    // event will be used to control the type of modal opening.
-    const { displayModal, toggleModal} = props;
-    toggleModal(!displayModal);
+    const modalType: string = e.currentTarget.name;
+
+    if (modalType === 'teamConfig') {
```

Comment:
if you get more than one modal type or foresee having more than one, should consider using an enum for the modal state instead of a boolean